data/securecontexts.json

```diff
@@ -1,3 +1,6 @@
 {
+  "impl": {
+    "chromestatus": 6021277022158848
```

Comment:
I believe you can also add `"edgestatus": "Secure Contexts"`, see corresponding entry in the [Edge status JSON export](https://raw.githubusercontent.com/MicrosoftEdge/Status/production/status.json).

Reply:
Thanks for the hint Francois - you are right as usual.